### PR TITLE
Make gas grid links bidirectional

### LINKED ISSUE #1021: Gas pipelines should be bidirectional
Gas grid links should be bidirectional, there p_nom_min should be -1 (instead of 0, the default value).

### PR DESCRIPTION
**Pull request by AmeliaNadal**
Fixes #1021.

## Before merging into `dev`-branch, please make sure that

- [x] the `CHANGELOG.rst` was updated.
- [x] new and adjusted code is formated using `black` and `isort`.
- [x] the `Dataset`-version is updated when existing datasets are adjusted.
- [x] the branch was merged into the
      `continuous-integration/run-everything-over-the-weekend`-branch.
- [x] the workflow is running successful in `test mode`.
- [x] the workflow is running successful in `Everything` mode.

### CODE REVIEW

File: src/egon/data/datasets/pypsaeursec/__init__.py [hunk 1]
```diff
@@ -743,6 +743,7 @@ def links_to_etrago(neighbor_links, scn="eGon100RE", extendable=True):
                     "efficiency4",
                     "lifetime",
                     "p_nom",
+                    # "p_min_pu",
```
Comment (IlkaCu) [inline src/egon/data/datasets/pypsaeursec/__init__.py:746]: Could you remove the lines which are commented out. Here and in line 756. Or are they needed somehow?

Comment (AmeliaNadal) [reply src/egon/data/datasets/pypsaeursec/__init__.py]: Sorry about that, I removed these lines.

**Review by IlkaCu — COMMENTED** [review]
You should also increase the version number in /gas_neighbours/__init__.py. Or am I mistaken?

**Comment by AmeliaNadal** [comment]
> You should also increase the version number in /gas_neighbours/**init**.py. Or am I mistaken?

Yes, you're fully right, I corrected this in https://github.com/openego/eGon-data/pull/1022/commits/e0d79e1b08af57c107f0efc882e51d2cd5727902

**Review by IlkaCu — APPROVED** [review]

**Pull request merged**